GitWorkflow/GitWorkflow/Player.cpp

```diff
@@ -64,11 +32,9 @@ void Player::update(const float dt)
 		m_hasJumped = true;
 	}
 
-	move(m_velocity * dt + sf::Vector2f(0.0f, 9.82f) * (dt * dt / 2.0f));
+	move(sf::Vector2f(0.0f, 9.82f) * (dt * dt / 2.0f));
 	m_velocity.y += 9.82f * dt;
-}
 
-void Player::draw(sf::RenderTarget& target, sf::RenderStates states) const
-{
-	target.draw(m_model, states);
+	GameObject::update(dt);
```

Comment:
I personally don't like two movement instructions (move + GameObject::update()), but it is the simplest solution for now.